README.md

```diff
@@ -34,30 +33,24 @@ See the [issue milestones](https://github.com/OrchardCMS/OrchardCore/milestones)
 
 ## Getting Started and Documentation
 
-The documentation can be accessed under <https://docs.orchardcore.net/>. See [the getting started docs](https://docs.orchardcore.net/en/latest/docs/getting-started/) on how to start using Orchard Core.
-
-You can also run Orchard Core from Docker:
-
-```
-docker run --name orchardcms -p 8080:80 orchardproject/orchardcore-cms-linux:latest
-```
-
-Docker images and parameters can be found at <https://hub.docker.com/u/orchardproject/>. See [our Docker documentation](https://docs.orchardcore.net/en/latest/docs/topics/docker/) for more details, especially if you're new to Docker.
+The documentation can be accessed under <https://docs.orchardcore.net/>. See the homepage for an overview, and [the getting started docs](https://docs.orchardcore.net/en/latest/docs/getting-started/) on how to start building apps with Orchard Core. If you'd just like to test drive Orchard Core as a user, check out [Test drive Orchard Core](https://docs.orchardcore.net/en/latest/docs/getting-started/test-drive-orchard-core/).
 
 ## Help and Support
 
 Do you need some help with Orchard Core? Don't worry, there are ways to get help from the community:
 
 - Did you find a bug or have a feature request? Open an issue [in the issue tracker](https://github.com/OrchardCMS/OrchardCore/issues).
 - Do you have a question about how to do something with Orchard Core, or would like a second opinion on your code? Open [a discussion](https://github.com/OrchardCMS/OrchardCore/discussions).
+- Do you want to chat with other community members? Check out [our Discord server](https://discord.gg/s3e2HtyPZc) and [Gitter chatroom](https://gitter.im/OrchardCMS/OrchardCore]).
 
 ## Get in Touch
 
 - [X (Twitter)](https://twitter.com/orchardcms)
 - [LinkedIn](https://www.linkedin.com/groups/13605669/)
-- [Meta (Facebook)](https://www.facebook.com/OrchardCore)
+- [Facebook](https://www.facebook.com/OrchardCore)
```

Comment:
```suggestion
- [Meta (Facebook)](https://www.facebook.com/OrchardCore)
```

just like Twittter

Reply:
The product is still called Facebook, even though the parent company is Meta. If you go to facebook.com, everything is still "Facebook", on contrary to Twitter.

Reply:
then maybe we should not have relabeled everything in from Facebook to Meta in the FB modules

Reply:
[Meta Pixel](https://www.facebook.com/business/tools/meta-pixel) is actually called "Meta". But renaming the login feature may have been premature indeed, since that's still [Facebook Login](https://developers.facebook.com/products/facebook-login/).

Reply:
moving on...